src/main/java/com/github/getchoo/smoothboot/mixin/UtilMixin.java

```diff
@@ -75,7 +72,7 @@ private static ExecutorService replWorker(String name) {
 
 		return new ForkJoinPool(MathHelper.clamp(select(name, SmoothBoot.config.threadCount.bootstrap,
 			SmoothBoot.config.threadCount.main), 1, 0x7fff), (forkJoinPool) -> {
-				String workerName = "Worker-" + name + "-" + NEXT_WORKER_ID.getAndIncrement();
+				String workerName = "Worker-" + name + "-" + new AtomicInteger(1).getAndIncrement();
```

Comment:
This will make it use the same ID (1) for every thread. The `AtomicInteger` should be moved outside the lambda to keep a counter.

Reply:
thanks!